src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -990,4 +991,17 @@ private Object readResolve() {
             return cloud.getTemplates();
         }
     }
+
+    @Initializer(after = InitMilestone.SYSTEM_CONFIG_LOADED)
+    public static void hpiRunInit() {
+        if (Main.isDevelopmentMode) {
```

Comment:
This does not trigger in `JenkinsRule`, right?

Reply:
You made me doubt, but it works as advertised (When using `JenkinsRule`, `Main.isUnitTest == true` but `Main.isDevelopmentMode == false`)